README.md

```diff
@@ -5,11 +5,12 @@ node.js command line bitcoin HD wallet, BIP 44 compatible.
 
 ## Requirements
 
-### NPM modules
+Until we have posted the module in the NPM registry, for installing it globally using the command `npm install -g waljs`,
+we must clone the repository and execute `npm install` and then `npm link`, then will be able to use it directly in the cmd.
```

Comment:
Rephrase to speak to the user/reader of the repo and not to ourselves: just give instructions on how to install and run.